src/qforte/system/molecular_info.py

```diff
@@ -162,3 +162,34 @@ def ccsd_energy(self):
     def ccsd_energy(self, ccsd_energy):
         self._ccsd_energy = ccsd_energy
 
+    @property
```

Comment:
Please define what these variables are supposed to be (especially expected types) in a docstring.